test/prototype/test_quantized_training.py

```diff
@@ -188,6 +188,7 @@ def snr(ref, actual):
         assert snr(inputs_ref.grad, inputs_int8mp.grad) > 20
         assert snr(linear.weight.grad, linear_int8mp.weight.grad) > 20
 
+    @pytest.mark.skip('Flaky on CI')
```

Comment:
should only skip if the version of pytorch is at least 2.6, the other tests seemed green